icu4c/source/common/unicode/localematcher.h

```diff
@@ -20,26 +20,24 @@
  * \brief C++ API: Locale matcher: User's desired locales vs. application's supported locales.
  */
 
-#ifndef U_FORCE_HIDE_DRAFT_API
-
 /**
  * Builder option for whether the language subtag or the script subtag is most important.
  *
- * @see LocaleMatcher::Builder#setFavorSubtag(ULocMatchFavorSubtag)
- * @draft ICU 65
+ * @see Builder#setFavorSubtag(ULocMatchFavorSubtag)
```

Comment:
Looks like you didn't merge, you are now just clobbering my earlier changes. Please fix.

Reply:
Sorry; I should have paid better attention.  Let me try to figure out how to get those changes back now...

Reply:
For reference: PR #1368

Reply:
Thanks for that; hopefully fixed now.  Apologies once again for not looking more closely.